src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -192,3 +202,59 @@
      [status-bar/status-bar {:type :modal}]
      [toolbar-view view-id unsigned-transactions]
      [main-section view-id tabs]]))
+
+(defn transaction-details-list-row [label value]
+  [react/view {:style transactions.styles/transaction-details-row}
+   [react/text {:style transactions.styles/transaction-details-item-label} (i18n/label label)]
+   [react/text {:style transactions.styles/transaction-details-item-value} (str value)]])
+
+(defn transaction-details-header [{:keys [value date type]}]
+  [react/view {:style transactions.styles/transaction-details-header}
+   [react/view {:style transactions.styles/transaction-details-header-icon}
+    [list/item-icon (transaction-type->icon type)]]
+   [react/view {:style transactions.styles/transaction-details-header-infos}
+    [react/text {:style transactions.styles/transaction-details-header-value} (str value " ETH")]
+    [react/text {:style transactions.styles/transaction-details-header-date} date]]])
+
+(defn transaction-details-confirmations [confirmations]
+  [react/view {:style transactions.styles/transaction-details-block}
+   [react/view {:style transactions.styles/transaction-details-confirmations-progress-bar}
+    [react/view {:style (transactions.styles/transaction-details-confirmations-progress-bar-progress confirmations)}]
+    [react/view {:style (transactions.styles/transaction-details-confirmations-progress-bar-total confirmations)}]]
+   [react/text {:style transactions.styles/transaction-details-confirmations-count}
+    (str confirmations " " (i18n/label :t/confirmations))]
+   [react/text {:style transactions.styles/transaction-details-confirmations-helper-text}
+    (i18n/label :t/confirmations-helper-text)]])
+
+(defn transaction-details-list [{:keys [block hash from to gas-limit gas-price gas-used cost nonce data]}]
+  [react/view {:style transactions.styles/transaction-details-block}
+   [transaction-details-list-row :t/block block]
```

Comment:
Would it be worth having a higher level component that would take a map? Can this component be reused in some other screen?

Reply:
hard to say regarding higher level component because there will be some edge cases, it's not fully complete yet since from/to for instance should show the wallet in black and address in gray for the known addresses.

I haven't seen an other screen that needs such compoment